javatests/arcs/android/entity/BUILD

```diff
@@ -0,0 +1,43 @@
+load(
+    "//third_party/java/arcs/build_defs:build_defs.bzl",
+    "arcs_kt_android_test_suite",
+)
+
+licenses(["notice"])
+
+TEST_SRCS = glob(["*Test.kt"])
+
+arcs_kt_android_test_suite(
+    name = "entity",
+    srcs = TEST_SRCS,
```

Comment:
Just delete this line and the TEST_SRCS variable. The default is to glob all kt files